### PR TITLE
Publish snapshots

### PR DESCRIPTION
**Pull request by milessabin**

### CODE REVIEW

**Review by joroKr21 — APPROVED** [review]

File: build.sbt [hunk 1]
```diff
@@ -40,7 +40,7 @@ ThisBuild / githubWorkflowBuild := Seq(
 )
 
 ThisBuild / githubWorkflowTargetTags ++= Seq("v*")
-ThisBuild / githubWorkflowPublishTargetBranches :=
+ThisBuild / githubWorkflowPublishTargetBranches +=
```
Comment (joroKr21) [inline build.sbt:43]: 😄 
```suggestion
ThisBuild / githubWorkflowPublishTargetBranches ++=
```

Comment (milessabin) [reply build.sbt:43]: Yeah, I borked that completely :roll_eyes:

**Comment by milessabin** [comment]
Unclear why that didn't work. I haven't yet set up the secrets, so I was expecting this to try to publish and fail, but instead the job seems to have simply been skipped. I wonder if the job is implicitly conditional on the secrets being present?

**Comment by joroKr21** [comment]
> Unclear why that didn't work. I haven't yet set up the secrets, so I was expecting this to try to publish and fail, but instead the job seems to have simply been skipped. I wonder if the job is implicitly conditional on the secrets being present?

Ahh, it's a `master` vs `main` thing - we are still using `master` and `sbt-github-actions` generated `main`.

**Comment by milessabin** [comment]
Riiight ... that's also on my TODO list, but I wanted to do one thing at a time.

**Pull request merged**